MISC/SetCloudsAlpha.md

```diff
@@ -0,0 +1,45 @@
+---
+ns: MISC
+aliases: ["0xF36199225D6D8C86", "_SET_CLOUD_HAT_OPACITY"]
+---
+## SET_CLOUDS_ALPHA
+
+```c
+// 0xF36199225D6D8C86
+void SET_CLOUDS_ALPHA(float opacity);
+```
+
+Allows modification of the cloud intensity when the player is in a switch state [`IS_PLAYER_SWITCH_IN_PROGRESS`](#_0xD9D2CFFF49FAB35F).
+
+You can set the player in a switch state with [`SWITCH_TO_MULTI_FIRSTPART`](#_0xAAB3200ED59016BC).
```

Comment:
I'm afraid this allows the modification of the cloud's alpha at any given time, the player doesn't need to be switching states or anything alike.

I was coincidentally researching this and the cloud opacity is actually a global float that can be set via `SET_CLOUDS_ALPHA`.  That float is then read via a `CRenderPhaseDrawScene` method where it's added to a draw command data structure.

This is how I tested it:

```lua
RegisterCommand('cloudopacity', function(source, args)
    SetWeatherTypeNowPersist('CLOUDS')
    SetCloudHatOpacity(tonumber(args[1])) -- Original native name
end, false)
```